src/include/firebird/impl/inf_pub.h

```diff
@@ -171,6 +174,18 @@ enum db_info_crypt			/* flags set in fb_info_crypt_state */
 	fb_info_crypt_process = 0x02
 };
 
+enum info_provider_features // response to fb_info_provider_features
+{
+	fb_feature_multi_statements 	= 0,    // Multiple prepared statements in single attachment
+	fb_feature_multi_transactions	= 1,	// Multiple concurrent transaction in single attachment
+	fb_feature_named_parameters		= 2,	// Query parameters can be named
+	fb_feature_session_reset		= 3,	// ALTER SESSION RESET is supported
+	fb_feature_read_consistency		= 4,	// Read consistency TIL is supported
+	fb_feature_statement_timeout	= 5,	// Statement timeout is supported
+
```

Comment:
While we don't have any rules engraved in stone in this regard, I'd prefer the numbering started with 1.

Reply:
As a trap for bugs, I see.